_posts/2025-04-28-sparse-autodiff.md

```diff
@@ -418,16 +431,15 @@ Mirroring the diversity of existing approaches to AD,
 there are also many possible approaches to sparsity pattern detection,
 each with their own advantages and tradeoffs.
 
-The method we will present here corresponds to a binary forward-mode AD system 
-in which performance is gained by compressing matrix rows.
```

Comment:
I prefer not to use the term "compress" here because compression is already used to describe things related to coloring (especially as opposed to decompression).
If you find another word to describe this, feel free to add it again.